### ISSUE TITLE
create plugin failed with gradle

### ISSUE THREAD

**Issue by windows7lake** [JetBrains/gradle-intellij-plugin#703]
After create plugin failed with gradle, following https://plugins.jetbrains.com/docs/intellij/gradle-prerequisites.html#components-of-a-wizard-generated-gradle-intellij-platform-plugin.
And it report that
```
Build file '/Users/yinglin/Workspace/JavaWorker/untitled/build.gradle' line: 22

A problem occurred evaluating root project 'untitled'.
> No signature of method: build_69rs5i4y7745cu3tjj97xlvk9.patchPluginXml() is applicable for argument types: (build_69rs5i4y7745cu3tjj97xlvk9$_run_closure4) values: [build_69rs5i4y7745cu3tjj97xlvk9$_run_closure4@545fab1a]

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

The project can not build.

**Comment by windows7lake** [comment]
build.gradle:
```
plugins {
    id 'java'
    id 'org.jetbrains.intellij' version '1.0'
}

group 'org.example'
version '1.0-SNAPSHOT'

repositories {
    mavenCentral()
}

dependencies {
    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
}

// See https://github.com/JetBrains/gradle-intellij-plugin/
intellij {
    version '2021.1.2'
}
patchPluginXml {
    changeNotes """
      Add change notes here.<br>
      <em>most HTML tags may be used</em>"""
}
test {
    useJUnitPlatform()
}
```

**Comment by hsz** [comment]
Unfortunately, the template used by Wizard, bundled within the IDE uses an incorrect assignment – without the `=` sign.
That has already been fixed in the upcoming 2021.2, but for now please adjust the `intellij` and `patchPluginXml` closures in a way described in the linked documentation page.

**Comment by calarasmihail** [comment]
@windows7lake 

Before:

```
intellij {
    version '2021.1'
}
patchPluginXml {
    changeNotes """
      Add change notes here.<br>
      <em>most HTML tags may be used</em>"""
}
```

After:

```
intellij {
    version = '2021.1'
}
patchPluginXml {
    changeNotes = """
      Add change notes here.<br>
      <em>most HTML tags may be used</em>"""
}
```
